pkgs/applications/networking/instant-messengers/signal-desktop/default.nix

```diff
@@ -147,7 +147,7 @@ in stdenv.mkDerivation rec {
     homepage    = "https://signal.org/";
     changelog   = "https://github.com/signalapp/Signal-Desktop/releases/tag/v${version}";
     license     = lib.licenses.agpl3Only;
-    maintainers = with lib.maintainers; [ ixmatus primeos equirosa ];
+    maintainers = with lib.maintainers; [ ];
```

Comment:
```suggestion
    maintainers = with lib.maintainers; [ mic92 ];
```

Reply:
I signed up to github releases just now. I am actively using signal.

Reply:
Awesome, thanks a lot! I'm glad that it will be in good hands :)